src/desktop/index.js

```diff
@@ -2,86 +2,103 @@ import { data as sd } from "sharify"
 
 const app = (module.exports = require("express")())
 
-// NOTE:
-// App order matters as some apps establish logic that is shared inside of subapps.
+// FIXME: Remove this pattern once bucket-assets is removed
+boot()
 
-// Apps with hardcoded routes or "RESTful" routes
-app.use(require("./apps/home"))
-app.use(require("./apps/editorial_features"))
-app.use(require("./apps/apply"))
-app.use(require("./apps/auctions"))
-app.use(require("./apps/auctions2"))
-app.use(require("./apps/auction_lots"))
-app.use(require("./apps/artist/server").app)
-app.use(require("./apps/artists"))
-app.use(require("./apps/auction"))
-app.use(require("./apps/auction_support"))
-app.use(require("./apps/artwork"))
-app.use(require("./apps/artwork2/server").app)
-app.use(require("./apps/about"))
-app.use(require("./apps/collect_art"))
-app.use(require("./apps/collect2/server").app)
-app.use(require("./apps/collections/server").app)
-app.use(require("./apps/categories"))
-app.use(require("./apps/consign"))
-app.use(require("./apps/contact"))
-app.use(require("./apps/eoy_2016"))
-app.use(require("./apps/how_auctions_work"))
-app.use(require("./apps/inquiry"))
-app.use(require("./apps/fairs"))
-app.use(require("./apps/partnerships"))
-app.use(require("./apps/gene"))
-app.use(require("./apps/geo"))
-app.use(require("./apps/jobs"))
-app.use(require("./apps/notifications"))
-app.use(require("./apps/order2/server.js"))
-app.use(require("./apps/personalize"))
-app.use(require("./apps/press"))
-app.use(require("./apps/search"))
-app.use(require("./apps/show"))
-app.use(require("./apps/shows"))
-app.use(require("./apps/tag"))
-app.use(require("./apps/unsubscribe"))
-app.use(require("./apps/unsupported_browser"))
-app.use(require("./apps/style_guide"))
-if (sd.NEW_AUTH_MODAL) {
-  app.use(require("./apps/auth2"))
-} else {
-  app.use(require("./apps/auth"))
-}
-app.use(require("./apps/static"))
-app.use(require("./apps/clear_cache"))
-app.use(require("./apps/sitemaps"))
-app.use(require("./apps/rss"))
-app.use(require("./apps/dev"))
-app.use(require("./apps/article"))
-app.use(require("./apps/artsy_primer"))
-app.use(require("./apps/gallery_partnerships"))
-app.use(require("./apps/marketing_signup_modals"))
-app.use(require("./apps/artsy_in_miami").default)
-app.use(require("./apps/armory_week").default)
-app.use(require("./apps/frieze_week").default)
-app.use(require("./apps/frieze_week_london").default)
-app.use(require("./apps/basel_week").default)
+function boot() {
+  if (process.env.ENABLE_EXPERIMENTAL_ASSET_BUNDLING) {
```

Comment:
Wanted a fairly representative app experience while i'm building out the new asset pipeline so gathered up all of the new apps under this flag. The `else` branch contains whats currently in master.